modules/channel-web/assets/examples/embedded-webchat.html

```diff
@@ -10,13 +13,13 @@
 
   <script>
     window.botpressWebChat.init({
-      host: 'http://localhost:3000',
+      host: 'http://localhost:3000/',
```

Comment:
webchat's init function should add the trailing `/` to make it backward compatible and also `host` normally doesn't take a protocol (http), so should probably be renamed `baseURL` in the next version

Reply:
I got the good balance to get everything to work by using a single source of truth (the external URL path) with those changes. When you are on the studio, the host is not set and we load the page with its relative url (assets/bla), but it works since the base href is set by the lite view. 

We could force setting the host on the studio tho.. and change it to baseURL instead. I didn't check if it may impact something else.